.github/workflows/continuous-integration.yml

```diff
@@ -202,9 +202,14 @@ jobs:
   contract-tests:
     name: Contract Tests
     runs-on: ubuntu-latest
+    outputs:
```

Comment:
@department-of-veterans-affairs/qa-standards I tagged you all as reviewers because of the changes to the contract test jobs